### ISSUE TITLE
Lower than benchmark accuracy for running Keras mnist_mlp example

### ISSUE THREAD

**Issue by gsimchoni** [fchollet/keras#4277]
I'm using Keras over TensorFlow (CPU version), Anaconda installation of python 3.5 on Linux Ubuntu 16.04 in Oracle VM VirtualBox, in Windows 7.

I'm running the examples/mnist_mlp.py file as is for classifying the MNIST dataset:

    $python examples/mnist_mlp.py
I'm doing this both from the Terminal and in Pycharm.

Everything runs fine, however I'm getting Test accuracy of 74.88% in both cases, despite the benchmark in the file saying I should get 98.40% (and obviously even SVM or Random Forests can easily pass 90% accuracy for MNIST).

What could have gone wrong? Remember I'm running the script as is, not changing any parameters or seed.
